FpsCheck.py

```diff
@@ -469,32 +452,29 @@ def find_invalid_alias_eSLDs(self, check_sets):
             None
         """
         for primary, curr_set in check_sets.items():
-            if not curr_set.ccTLDs:
-                continue
-            for aliased_site in curr_set.ccTLDs:
+            for alias, aliases in curr_set.ccTLDs.items():
```

Comment:
I think this should also be `aliased_site, aliases`

Reply:
Done.